strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedRunnerProperties.java

```diff
@@ -20,9 +25,23 @@
     static final String EXCLUSIONS = "exclusions";
 
     private final Properties properties = new Properties();
+    private AffectedConfiguration affectedConfiguration;
 
-    AffectedRunnerProperties(){
-        readFile(System.getProperty(SMART_TESTING_AFFECTED_CONFIG));
+    AffectedRunnerProperties(File rootDirectory) {
+        final Configuration configuration = Configuration.loadPrecalculated(rootDirectory);
+        final List<StrategyConfiguration> strategyConfigurations = configuration.getStrategiesConfiguration();
+        if (strategyConfigurations == null) {
+            readFile(null);
+            return;
+        }
+
+        final StrategyConfiguration affectedConfig = strategyConfigurations.stream()
+            .filter(strategyConfiguration -> "affected".equals(strategyConfiguration.name()))
```

Comment:
> "affected"

I would use a constant or method that returns the name of the strategy

Reply:
We are using it only one. So I haven't used any constant.

Reply:
second usage: https://github.com/arquillian/smart-testing/blob/master/strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedChangesDetectorFactory.java#L13